Perl/Perl.sublime-syntax

```diff
@@ -1213,7 +1213,7 @@ contexts:
     - match: \.=?
       scope: keyword.operator.concatenation.perl
       push: expressions-begin
-    # string concation
+    # string repetition
```

Comment:
notable.

alternatively `concatenation`, but `x` really isn't concatenation, `.` is concatenation.

Reply:
This was discussed in https://github.com/sublimehq/Packages/pull/4127#discussion_r1900933804